### PR TITLE
PMM-10393 Add MongoDB collector scrape time

### PR DESCRIPTION
**Pull request by ShashankSinha252**
PMM-10393 : Added new panel for showing scrape time of MongoDB collectors

Feature build: SUBMODULES-2757

MongoDB Collector Scrape Time

### CODE REVIEW

**Review by BupycHuk — DISMISSED** [review]

File: dashboards/Insight/Prometheus_Exporter_Status.json [hunk 1]
```diff
@@ -51,8 +51,8 @@
   "editable": false,
   "fiscalYearStartMonth": 0,
   "graphTooltip": 1,
-  "id": null,
-  "iteration": 1656404952623,
+  "id": 5,
```
Comment (anton-bystrov) [inline dashboards/Insight/Prometheus_Exporter_Status.json:54]: Here should be "null"

File: dashboards/Insight/Prometheus_Exporter_Status.json [hunk 2]
```diff
@@ -2058,13 +2170,18 @@
       "targets": [
         {
           "calculatedInterval": "10s",
+          "datasource": {
+            "type": "prometheus",
+            "uid": "PA58DA793C7250F1B"
+          },
```
Comment (anton-bystrov) [inline dashboards/Insight/Prometheus_Exporter_Status.json:2176]: Can you change datasource to "datasource": "Metrics" ?

File: dashboards/Insight/Prometheus_Exporter_Status.json [hunk 3]
```diff
@@ -4533,7 +4650,7 @@
       "yBucketBound": "auto"
     }
   ],
-  "refresh": "1m",
+  "refresh": "5s",
```
Comment (anton-bystrov) [inline dashboards/Insight/Prometheus_Exporter_Status.json:4653]: Default refresh time is 1m. Five second is too often.

File: dashboards/Insight/Prometheus_Exporter_Status.json [hunk 4]
```diff
@@ -4703,6 +4826,6 @@
   "timezone": "",
   "title": "Prometheus Exporter Status",
   "uid": "prometheus-status",
-  "version": 1,
+  "version": 2,
```
Comment (anton-bystrov) [inline dashboards/Insight/Prometheus_Exporter_Status.json:4829]: Here "version": 1,

File: dashboards/Insight/Prometheus_Exporter_Status.json [hunk 5]
```diff
@@ -4641,7 +4761,10 @@
           "text": "All",
           "value": "$__all"
         },
-        "datasource": "Metrics",
+        "datasource": {
+          "type": "prometheus",
+          "uid": "PA58DA793C7250F1B"
+        },
```
Comment (anton-bystrov) [inline dashboards/Insight/Prometheus_Exporter_Status.json:4767]: Change datasource please.

File: dashboards/Insight/Prometheus_Exporter_Status.json [hunk 6]
```diff
@@ -4610,10 +4727,13 @@
         "allFormat": "glob",
         "current": {
           "selected": false,
-          "text": "ip-10-178-0-240.us-east-2.compute.internal",
-          "value": "ip-10-178-0-240.us-east-2.compute.internal"
+          "text": "mongo",
+          "value": "mongo"
```
Comment (anton-bystrov) [inline dashboards/Insight/Prometheus_Exporter_Status.json:4731]: Can you change to "text": "",
                               "value": "" ?

**Review by anton-bystrov — CHANGES_REQUESTED** [review]

File: dashboards/Insight/Prometheus_Exporter_Status.json (hunk 1 above)
Comment (ShashankSinha252) [reply dashboards/Insight/Prometheus_Exporter_Status.json:54]: Done

File: dashboards/Insight/Prometheus_Exporter_Status.json (hunk 2 above)
Comment (ShashankSinha252) [reply dashboards/Insight/Prometheus_Exporter_Status.json:2176]: Done

File: dashboards/Insight/Prometheus_Exporter_Status.json (hunk 3 above)
Comment (ShashankSinha252) [reply dashboards/Insight/Prometheus_Exporter_Status.json:4653]: Done

File: dashboards/Insight/Prometheus_Exporter_Status.json (hunk 6 above)
Comment (ShashankSinha252) [reply dashboards/Insight/Prometheus_Exporter_Status.json:4731]: Done

File: dashboards/Insight/Prometheus_Exporter_Status.json (hunk 4 above)
Comment (ShashankSinha252) [reply dashboards/Insight/Prometheus_Exporter_Status.json:4829]: Done

**Review by anton-bystrov — APPROVED** [review]

**Review by BupycHuk — DISMISSED** [review]

File: dashboards/Insight/Prometheus_Exporter_Status.json [hunk 7]
```diff
@@ -4610,10 +4724,13 @@
...
         "current": {
           "selected": false,
-          "text": "ip-10-178-0-240.us-east-2.compute.internal",
-          "value": "ip-10-178-0-240.us-east-2.compute.internal"
+          "text": "",
+          "value": ""
+        },
+        "datasource": {
+          "type": "prometheus",
+          "uid": "PA58DA793C7250F1B"
```
Comment (denisok) [inline dashboards/Insight/Prometheus_Exporter_Status.json:4732]: @anton-bystrov is this a correct datasource?

Comment (anton-bystrov) [reply dashboards/Insight/Prometheus_Exporter_Status.json:4732]: No. Sorry, I didn't notice it.

Comment (denisok) [reply dashboards/Insight/Prometheus_Exporter_Status.json:4732]: @anton-bystrov could you please suggest correct changes or commit here?

**Review by denisok — CHANGES_REQUESTED** [review]

**Review by denisok — APPROVED** [review]

**Pull request merged**